Nos/Views/DiscoverView.swift

```diff
@@ -197,6 +200,17 @@ struct DiscoverView: View {
             .navigationBarItems(leading: SideMenuButton())
         }
     }
+
+    var notFindingResultsView: some View {
```

Comment:
I think it would be nice to show the progress indicator in this view. It reinforces the idea that we are still listening for relays to give us results.

Reply:
I think removing the blur is fine. In terms of 10 sec - I think we need to train people to wait a bit longer on Nostr as we don't store anything locally and we need to reach out to the relays to retrieve.

Reply:
Done! (moved to `FullscreenProgressView`)